changelog/bugfixes/2024-03-05-ensure-sysext-contents-merge.md

```diff
@@ -0,0 +1 @@
+- Ensured that systemd-sysext images can extend directories for which Flatcar extensions also ship files, e.g., that the sysext-bakery Kubernetes extension works when OEM extensions are present ([sysext-bakery#50](https://github.com/flatcar/sysext-bakery/issues/50))
```

Comment:
```suggestion
- Ensure that systemd-sysext images can extend directories where Flatcar extensions are also shipping files, e.g., that the sysext-bakery Kubernetes extension works when OEM extensions are present ([sysext-bakery#50](https://github.com/flatcar/sysext-bakery/issues/50))
```

Reply:
`Ensure` sounds like an imperative for actions the user should follow, I would keep `Ensured` or reword it to something like `Fixed`

Reply:
Fixed directories provided by Flatcar extensions to be extensible by other systemd-sysext images, e.g. Kubernetes yadda yadda?